code/game/machinery/doors/airlock.dm

```diff
@@ -793,7 +793,52 @@
 			return
 	add_fingerprint(user)
 
-	if(panel_open)
+	if(is_wire_tool(C) && panel_open)
+		attempt_wire_interaction(user)
+		return
+	else if(istype(C, /obj/item/pai_cable))
+		var/obj/item/pai_cable/cable = C
+		cable.plugin(src, user)
+	else if(istype(C, /obj/item/airlock_painter))
+		change_paintjob(C, user)
+	else if(istype(C, /obj/item/door_seal)) //adding the seal
+		var/obj/item/door_seal/airlockseal = C
+		if(!density)
+			to_chat(user, span_warning("[src] must be closed before you can seal it!"))
+			return
+		if(seal)
+			to_chat(user, span_warning("[src] has already been sealed!"))
+			return
+		user.visible_message(span_notice("[user] begins sealing [src]."), span_notice("You begin sealing [src]."))
+		playsound(src, 'sound/items/jaws_pry.ogg', 30, TRUE)
+		if(!do_after(user, airlockseal.seal_time, target = src))
+			return
+		if(!density)
+			to_chat(user, span_warning("[src] must be closed before you can seal it!"))
+			return
+		if(seal)
+			to_chat(user, span_warning("[src] has already been sealed!"))
+			return
+		if(!user.transferItemToLoc(airlockseal, src))
+			to_chat(user, span_warning("For some reason, you can't attach [airlockseal]!"))
+			return
+		playsound(src, 'sound/machines/airlockforced.ogg', 30, TRUE)
+		user.visible_message(span_notice("[user] finishes sealing [src]."), span_notice("You finish sealing [src]."))
+		seal = airlockseal
+		modify_max_integrity(max_integrity * AIRLOCK_SEAL_MULTIPLIER)
+		update_appearance()
+
+	else if(istype(C, /obj/item/paper) || istype(C, /obj/item/photo))
+		if(note)
+			to_chat(user, span_warning("There's already something pinned to this airlock! Use wirecutters to remove it."))
+			return
+		if(!user.transferItemToLoc(C, src))
+			to_chat(user, span_warning("For some reason, you can't attach [C]!"))
+			return
+		user.visible_message(span_notice("[user] pins [C] to [src]."), span_notice("You pin [C] to [src]."))
+		note = C
+		update_appearance()
+	else if(panel_open)
```

Comment:
Rather then making this an else if, I think it'd be cleaner to change
`else if(is_wire_tool(C) && panel_open)` to `if(is_wire_tool(C) && panel_open)
and keep the panel open stuff at the top

It better communicates its status as something that runs regardless of the other if checks

Reply:
will do in morning

Reply:
aight, sounds good. make sure you update your changelog to be more clear too yeah?

Reply:
what in fucks name i don’t even remember writing that changelog